src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/dto/orchestration/OrchestrationRejectVisitRequestResponseDto.kt

```diff
@@ -0,0 +1,18 @@
+package uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.dto.orchestration
+
+import io.swagger.v3.oas.annotations.media.Schema
+import jakarta.validation.constraints.NotBlank
+
+data class OrchestrationRejectVisitRequestResponseDto(
```

Comment:
can we rename OrchestrationApproveVisitRequestResponseDto to say OrchestrationVisitRequestResponseDto  and reuse it as it has the same fields? might easier for the front-end to handle.